### ISSUE TITLE
Request for source of Official SDK of Unionpay

### ISSUE THREAD

**Issue by nicktunac** [wangfei0904306/unionpay#1]
Hi wangfei,

Could you help me the source of the Official SDK of Unionpay for 5.1.0 where you used in this repository.

Currently we are using com.unionpay.acp.upacp-sdk:1.0.0, but it doesnt support SHA256 signing method 
http://maven.marsorstudio.cn/com/unionpay/acp/upacp-sdk/1.0.0/

Thank you.

**Comment by wangfei0904306** [comment]
https://open.unionpay.com/ajweb/product/newProDetail?proId=66&cataId=72

try it~

**Comment by nicktunac** [comment]
Thanks, @wangfei0904306 

Got it working already. But the problem I'm facing right now is when this project is packaged. It will throw an error that I cannot read the cert files.

I'm thinking of modifying the initCert() method in CertUtil class to read the cert files.

**Comment by wangfei0904306** [comment]
You are right, you can use the absolute path first, and if you succeed, put it in the appropriate location of the project.
And you may also face this problem if you package it with springboot jar, which you should resolve
